test_runner/regress/test_change_pageserver.py

```diff
@@ -23,19 +21,6 @@ def test_change_pageserver(neon_env_builder: NeonEnvBuilder, make_httpserver):
     )
     env = neon_env_builder.init_start()
 
-    neon_env_builder.control_plane_hooks_api = (
```

Comment:
how are we setting `use_local_compute_notifications` here? I think we want the test to disable it.

Reply:
The assignment has no effect since it's done after `init_start`. It was unused, so I removed it.
We set use_local_compute_notifications via neon local default.